### PR TITLE
Cu 39w8aku rename postgressql

### PR DESCRIPTION
**Pull request by MaximeWrlz**

### CODE REVIEW

**Review by ManonFargues — DISMISSED** [review]

File: src/modules/app.module.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Logger, Module } from '@nestjs/common';
 import { ConfigModule } from '@nestjs/config';
 import { DataSource } from 'typeorm';
-import { LoggerInterceptor } from './../interceptors/logger/logger.interceptor';
+import { LoggerInterceptor } from '../interceptors/logger/logger.interceptor';
```
Comment (FlavianEng) [inline src/modules/app.module.ts:4]: Add an alias path

**Review by FlavianEng — CHANGES_REQUESTED** [review]

**Review by FlavianEng — APPROVED** [review]

**Review by ManonFargues — APPROVED** [review]

**Pull request merged**